cloud-sql-proxy/README.MD

```diff
@@ -0,0 +1,49 @@
+# Cloud SQL I/O and Hive Metastore
+
+This initialization action installs a [Google Cloud SQL proxy](https://cloud.google.com/sql/docs/sql-proxy) on every node in a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. It also configures the cluster to store [Apache Hive](https://hive.apache.org) metadata on a given Cloud SQL instance.
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster using a shared hive metastore.:
+
+1. Using the `gcloud` command to create a new 2nd generation Cloud SQL intance (or use a previously created instance)
+
+    ```bash
+    gcloud sql instances create <INSTANCE_NAME> \
+        --tier db-n1-standard1 \
```

Comment:
s/db-n1-standard1/db-n1-standard-1/

Otherwise the error is:

    ERROR: (gcloud.sql.instances.create) Invalid value for: db-n1-standard1 is not a valid value